### PR TITLE
fix: sentry source maps

### PR DESCRIPTION
**Pull request by vasco-santos**
Adds webpack plugin to add source maps for sentry usage

- Secrets were added to the repo

TODO:

- [x] Errors with invalid version

```
Reason | invalid character '/'
-- | --
Value | @web3-storage/api-1.0.1
```

NOTE: This needs to be tested in staging/production to guarantee everything is as expected

### CODE REVIEW

**Review by hugomrdias — APPROVED** [review]

File: .github/workflows/api.yml [hunk 1]
```diff
@@ -42,6 +42,8 @@ jobs:
         uses: web3-storage/wrangler-action@node16
         env:
           SENTRY_TOKEN: ${{ secrets.STAGING_SENTRY_TOKEN}}
+          SENTRY_TOKEN: ${{ secrets.STAGING_SENTRY_PROJECT}}
+          SENTRY_TOKEN: ${{ secrets.STAGING_SENTRY_UPLOAD}}
```
Comment (alanshaw) [inline .github/workflows/api.yml:46]:
```suggestion
          SENTRY_PROJECT: ${{ secrets.STAGING_SENTRY_PROJECT}}
          SENTRY_UPLOAD: ${{ secrets.STAGING_SENTRY_UPLOAD}}
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -127,6 +127,8 @@ jobs:
         uses: web3-storage/wrangler-action@node16
         env:
           SENTRY_TOKEN: ${{ secrets.PROD_SENTRY_TOKEN}}
+          SENTRY_TOKEN: ${{ secrets.PROD_SENTRY_PROJECT}}
+          SENTRY_TOKEN: ${{ secrets.PROD_SENTRY_UPLOAD}}
```
Comment (alanshaw) [inline .github/workflows/release.yml:131]:
```suggestion
          SENTRY_PROJECT: ${{ secrets.STAGING_SENTRY_PROJECT}}
          SENTRY_UPLOAD: ${{ secrets.STAGING_SENTRY_UPLOAD}}
```

File: packages/api/webpack.config.js [hunk 3]
```diff
@@ -1,20 +1,40 @@
+import dotenv from 'dotenv'
 import { createRequire } from 'module'
+import SentryWebpackPlugin from '@sentry/webpack-plugin'
+import { GitRevisionPlugin } from 'git-revision-webpack-plugin'
```
Comment (alanshaw) [inline packages/api/webpack.config.js:4]: Needs to be added to dependencies?

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: packages/api/webpack.config.js (hunk 3 above)
Comment (vasco-santos) [reply packages/api/webpack.config.js:4]: yes, I was playing it other options before PRing and removed, but did not put back

**Review by alanshaw — APPROVED** [review]

**Pull request merged**